apps/leo_gateway/priv/leo_gateway.conf

```diff
@@ -109,6 +109,13 @@ bucket_prop_sync_interval = 300
 ## wtperf is set to the same value.
 ## nfs.nfsd.wtmax = 5242880
 
+## NFSd's Readdir Entry Table Scan Interval (leo-project/leofs#612)
+## nfs.nfsd.readdir_scan_int = 1800
+
+## NFSd's Readdir Entry TTL (leo-project/leofs#612)
```

Comment:
Let me specify it and align the unit with other settings.
It seems ms is the common unit here? I am using second here though